package.json

```diff
@@ -25,5 +25,8 @@
     "require-dir": "^0.3.0",
     "sass-lint": "^1.10.2"
   },
-  "dependencies": {}
+  "scripts": {
+    "gulp": "gulp",
```

Comment:
npm scripts prepend the local `node_modules/.bin` to the PATH so it doesn't require gulp to be installed globally.